raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -1,541 +1,774 @@
 
+Powers of the Council to collect, analyse and disseminate environmental information 
 (173) The Council shall: [a] gather information on the environment and natural resources on the existing data; [b] subject to any other written law, have access to any data collection on the environment and natural resources; [c] analyse information relating to the environment and natural resources; [d] disseminate information to public and private users; [e] commission studies in demographic and trends impacting on environmental and development issues; [f] carry out public information and education campaigns in the field of environment; [g] exchange information relating to environment with Non-Governmental Organisations or any other regional and international organisations; [h] co-ordinate the management of environmental information with Sector Ministries; [i] advise the Minister on existing information gaps and needs; or [j] establish in consultation with Sector Ministries, guidelines and principles for the gathering, processing and dissemination of environmental information.
 
-(174) The Council shall establish and operate a Central Environmental Information System which may bring together any findings, data and statistics generated by both public and private institutions in the course of environmental observation and management. 
+Central Environmental Information System 
+(174) The Council shall establish and operate a which may bring together any findings, data and statistics generated by both public and private institutions in the course of environmental observation and management. 
```

Comment:
`operate a which` still needs to be replaced with `operate a Central Environmental Information System which`.